legal-api/src/legal_api/models/user.py

```diff
@@ -107,7 +109,9 @@ def create_from_jwt_token(cls, token: dict):
                 firstname=token.get(current_app.config.get('JWT_OIDC_FIRSTNAME'), None),
                 lastname=token.get(current_app.config.get('JWT_OIDC_LASTNAME'), None),
                 iss=token['iss'],
-                sub=token['sub']
+                sub=token['sub'],
+                idp_userid=token['idp_userid'],
+                login_source=token['loginSource']
```

Comment:
I'm not sure why a lot of the tests are failing but maybe in the `helper_create_jwt` function of `legal-api/tests/unit/services/utils.py` you do not include the `loginSource` for the token.  Based on the column definition for `loginSource` in the `users` table, this seems like a valid scenario.  

If this is the source of the test errors, you will need to be able to handle an incoming token that has no `login_source` property.

Reply:
thank you - missed that. fixed

Reply:
so the incoming token will always have the `login_source` property?

Reply:
currently loginSource is a 'hardcoded attribute' for idir/bcsc/bceid IDPs

Reply:
ok, sounds like it will always be there so L114(`login_source=token['loginSource']`) won't have a scenario where it will error out due to the property not been available in the token.